docs/user/02-logs.md

```diff
@@ -437,77 +452,67 @@ The record **after** applying the JSON parser:
+To detect and fix such situations, check the pipeline status and check out [Troubleshooting](#troubleshooting).
 
 ## Limitations
 
-Currently, there are the following limitations for LogPipelines that are served by Fluent Bit:
+- **Reserved Log Attributes**: The log attribute named `kubernetes` is a special attribute that's enriched by the `kubernetes` filter. When you use that attribute as part of your structured log payload, the metadata enriched by the filter are overwritten by the payload data. Filters that rely on the original metadata might no longer work as expected.
+- **Buffer Limits**: Fluent Bit buffers up to 1 GB of logs if a configured output cannot receive logs. The oldest logs are dropped when the limit is reached or after 300 retries.
+- **Throughput**: Each Fluent Bit Pod (each running on a dedicated Node) can process up to 10 MB/s of logs for a single LogPipeline. With multiple pipelines, the throughput per pipeline is reduced. The used logging backend or performance characteristics of the output plugin might limit the throughput earlier.
+- **Max Amount of Pipelines**: The maximum amount of LogPipeline resources is 5.
 
 ### Unsupported Mode
 <!--- unsupported mode is not part of Help Portal docs --->
 The `unsupportedMode` attribute of a LogPipeline indicates that you are using a `custom` filter and/or `custom` output. The Kyma team does not provide support for a custom configuration.
 
 ### Fluent Bit Plugins
 <!--- Fluent Bit Plugins is not part of Help Portal docs --->
+
 You cannot enable the following plugins, because they potentially harm the stability:
 
 - Kubernetes Filter
 - Rewrite_Tag Filter
 
-### Reserved Log Attributes
-
-The log attribute named `kubernetes` is a special attribute that's enriched by the `kubernetes` filter. When you use that attribute as part of your structured log payload, the metadata enriched by the filter are overwritten by the payload data. Filters that rely on the original metadata might no longer work as expected.
-
-### Buffer Limits
-
-Fluent Bit buffers up to 1 GB of logs if a configured output cannot receive logs. The oldest logs are dropped when the limit is reached or after 300 retries.
-
-### Throughput
-
-Each Fluent Bit Pod (each running on a dedicated Node) can process up to 10 MB/s of logs for a single LogPipeline. With multiple pipelines, the throughput per pipeline is reduced. The used logging backend or performance characteristics of the output plugin might limit the throughput earlier.
-
-### Max Amount of Pipelines
-
-The maximum amount of LogPipelines is 5.
-
 ## Troubleshooting
 
 ### No Logs Arrive at the Backend
 
-Cause: Incorrect backend endpoint configuration (for example, using the wrong authentication credentials) or the backend being unreachable.
+**Symptom**: No logs arrive at the backend. In the LogPipeline status, the `TelemetryFlowHealthy` condition has status **AllDataDropped**.
```

Comment:
In the guide below you use a bullet list for the two symptoms, I prefer it here as well